youslist.txt

```diff
@@ -2231,6 +2237,7 @@
 ||tracker.digitalcamp.co.kr^
 ||tracker.nodojs.nl^
 ||tracking.adweb.co.kr^
+||traders.co.kr/common/js/makePCookie.js
```

Comment:
If this is used in `traders.co.kr` only, then please also add `$domain=traders.co.kr`.

```suggestion
||traders.co.kr/common/js/makePCookie.js^$domain=traders.co.kr
```